src/main/java/org/dinosaur/foodbowl/domain/member/api/MemberController.java

```diff
@@ -0,0 +1,23 @@
+package org.dinosaur.foodbowl.domain.member.api;
+
+import lombok.RequiredArgsConstructor;
+import org.dinosaur.foodbowl.domain.member.application.MemberService;
+import org.dinosaur.foodbowl.domain.member.dto.DuplicateCheckResponse;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RequestMapping("/api/v1/members")
+@RestController
+public class MemberController {
+
+    private final MemberService memberService;
+
+    @PostMapping("/check-nicknames")
```

Comment:
중복된 닉네임이 존재하는지 여부를 조회한다는 의미에서 `GET`이 더 적절해보이는데 `POST`를 사용하신 이유가 있을까요? 🤔

Reply:
```suggestion
    @GetMapping("/check-nickname")
```
`URL`은 잘 찾아주신 것 같습니다👍 단일 닉네임을 확인하는 것이니 단수로 바꿔보는건 어떨까요?

Reply:
저도 nickname이 컬랙션이 아닌 리소스라 단수가 적절해보입니다!

Reply:
저도 `GET`이 적절한 것 같아서 `GET`으로 처음에 설계했다가,  RequestBody로 요청을 받아서 정규식을 적용해보려고 `POST`로 바꿨었습니다. 깜빡하고 롤백을 안했었네용 !

그런데 RequestBody로 받지 않으면 Pattern을 활용한 검증을 할 수 없습니다 !

그렇다면, 서버에서 클라이언트가 보내주는 닉네임 형식을 검증할 필요가 있을지 이야기 해보면 좋을 것 같습니다 !

Reply:
우창님이 말한대로 닉네임이 있는지 없는지 판별하는 로직이라 `GET`으로 볼 수 있을거 같아요

Reply:
그런 문제가 있었군요 .. ! 닉네임을 파라미터로 받아서 `@RequestParam`에서 검증을 해보는건 어떨까요?

Reply:
클래스단에 `@Validated`로 선언하니 `@RequestParam`을 검증할 수 있네요 !